src/dlsnode/util/aio/internal/Queue.d

```diff
@@ -1,137 +0,0 @@
-/*******************************************************************************
-
-    Links the elements into the list, making a queue. To be used with
-    AioScheduler. `Queue.Element` should be inherited from the actual classes
-    that will be queued by thread workers, and processed by the main thread.
-    The reason for linking all these objects together and forming a queue in
```

Comment:
Does this reasoning still apply and is it worth mentioning this somewhere else?

Reply:
No, this doesn't apply anymore. It was made like this previously so we didn't had to create another nogc linked list (we had to keep the GC out of it). However, `TreeQueue` implements exactly that, so this is outdated now.